packages/odyssey-react/src/components/List/List.module.scss

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  max-width: $max-line-length;
+  margin-block-start: 0;
+  margin-block-end: $type-margin;
+  margin-inline: 0;
+  padding-block: 0;
+}
+
+.unordered {
+  list-style-type: disc;
+}
+
+.ordered {
+  list-style-type: decimal;
+}
+
+.ordered,
+.unordered {
+  padding-inline-start: $spacing-m;
+  padding-inline-end: 0;
+  list-style-position: outside;
+
+  &:last-child {
+    margin-block-end: 0;
+  }
+}
+
+.item {
+  &:last-child {
+    margin-block-end: 0;
+  }
+}
+
+.description {
+  display: grid;
+  grid-gap: $spacing-xs $spacing-m;
+  grid-template-columns: repeat(2, minmax(min-content, max-content));
+  padding-inline: 0;
+
+  &:last-child {
+    margin-block-end: 0;
+  }
+}
+
+.term {
+  grid-column: 1/1;
+  font-weight: 600;
+}
+
+.details {
+  grid-column: 2/2;
+  font-weight: 400;
+}
+
+.unstyled {
+  list-style: none;
+  margin-block: 0;
+  padding-inline-start: 0;
+}
```

Comment:
beautiful css module styles here :tada: low specificity and easy to reason about